### PR TITLE
stdoutlogger

### PR DESCRIPTION
**Pull request by crewtool**
# Description
Added stdout logger
[neptune](https://app.neptune.ai/o/pmtest/org/llm-random/runs/details?viewId=standard-view&detailsTab=metadata&shortId=LLMRANDOM-8306&type=run) and [wandb](https://wandb.ai/ideas_cv/llm-random/runs/1vyp6sog?nw=nwusercrewtool) are the same

# Checklist
   - [ ] Is this PR focused on a single feature without the possibility of being divided into smaller PRs?
   - [ ] Are all variables set with logical defaults that are backward compatible?
   - [ ] I attached link to neptune if it was necessary
   - [ ] Have you successfully executed the linter and tests?
   - [ ] Are all functions concise and don't require splitting?
   - [ ] Is there documentation for arguments and complex logic?
   - [ ] Are there no temporary solutions in the code? Should a task be added to Trello to refine the code?
   - [ ] Are all functions placed in appropriate files?
   - [ ] Is the PR name meaningful and descriptive?
   - [ ] Is PR description provided, or unnecessary?
  
Other questions:
   - [ ] If I made changes to the requirements or anything related to the Singularity image I did generate new image and upload a new image to the clusters
   - [ ] If this change is significant enough I notified all individuals working with this repository (@channel in slack channel)
   - [ ] If there is a need for a second reviewer I requested additional review 

# Reviewer's checklist:
   - [ ] Does the Neptune link work and does it actually compare the situation before and after the PR?
   - [ ] Do I understand the code?
   - [ ] Do I think another reviewer is not needed or I commented that this PR need another reviewer?

### CODE REVIEW

File: configs/olaaa.yaml
Comment (maciek-pioro) [inline configs/olaaa.yaml:1]: del

File: lizrd/support/logging.py [hunk 1]
```diff
@@ -329,49 +431,46 @@ def log_plot(figure: plotly.graph_objs.Figure, title: str, series: str, iteratio
...
+    if args.logger_types == "":
+        logger_types = []
     else:
-        print(
-            "No logger specified! either args.use_neptune / args.use_clearml / args.use_wandb must be True"
-        )
-        raise NotImplementedError
+        logger_types = args.logger_types.split(",")
+    initialized_loggers = []
+    for logger_type in logger_types:
```
Comment (maciek-pioro) [inline lizrd/support/logging.py:439]: Maybe let's add validation that doesn't allow specifying the same type twice? e.g. "neptune,stdout,neptune"

**Review by maciek-pioro — CHANGES_REQUESTED** [review]
Minor changes, also let's wait a couple of hours and see if people are not furious about removing clearml

File: lizrd/support/logging.py (hunk 1 above)
Comment (crewtool) [reply lizrd/support/logging.py:439]: Good catch!

**Review by maciek-pioro — APPROVED** [review]

**Comment by maciek-pioro** [comment]
LGTM

**Pull request merged**